<Fault xmlns="http://schemas.microsoft.com/ws/2005/05/envelope/none"><Code><Value>Service</Value></Code><Reason><Text xml:lang="en-US">Sequence contains no elements</Text></Reason><Detail><InvalidOperationException z:Id="1" z:Type="System.InvalidOperationException" z:Assembly="0" xmlns="http://schemas.datacontract.org/2004/07/System" xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns:x="http://www.w3.org/2001/XMLSchema" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><ClassName z:Id="2" z:Type="System.String" z:Assembly="0" xmlns="">System.InvalidOperationException</ClassName><Message z:Id="3" z:Type="System.String" z:Assembly="0" xmlns="">Sequence contains no elements</Message><Data z:Id="4" z:Type="System.Collections.ListDictionaryInternal" z:Assembly="0" xmlns=""><count xmlns="http://schemas.microsoft.com/2003/10/Serialization/Arrays">0</count><head i:nil="true" xmlns="http://schemas.microsoft.com/2003/10/Serialization/Arrays" xmlns:a="http://schemas.datacontract.org/2004/07/System.Collections"/><version xmlns="http://schemas.microsoft.com/2003/10/Serialization/Arrays">0</version></Data><InnerException i:nil="true" xmlns=""/><HelpURL i:nil="true" xmlns=""/><StackTraceString z:Id="5" z:Type="System.String" z:Assembly="0" xmlns=""> at System.Linq.Enumerable.First[TSource](IEnumerable`1 source)&#xD; at WebServiceLib.Webservice.AuthHelper.GetVerifiedUserID(ClaimsIdentity identity)&#xD; at WebServiceLib.Webservice.ResponseHelper.GenerateErrorResponse(HttpStatusCode statusCode, String errorMessage)&#xD; at Auth.Webservice.AuthService.GetToken(LoginParams loginParams)&#xD; at SyncInvokeLogin(Object , Object[] , Object[] )&#xD; at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]&amp; outputs)&#xD; at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc&amp; rpc)&#xD; at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc&amp; rpc)&#xD; at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc&amp; rpc)&#xD; at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)</StackTraceString><RemoteStackTraceString i:nil="true" xmlns=""/><RemoteStackIndex z:Id="6" z:Type="System.Int32" z:Assembly="0" xmlns="">0</RemoteStackIndex><ExceptionMethod z:Id="7" z:Type="System.String" z:Assembly="0" xmlns="">8 First System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089 System.Linq.Enumerable TSource First[TSource](System.Collections.Generic.IEnumerable`1[TSource])</ExceptionMethod><HResult z:Id="8" z:Type="System.Int32" z:Assembly="0" xmlns="">-2146233079</HResult><Source z:Id="9" z:Type="System.String" z:Assembly="0" xmlns="">System.Core</Source><WatsonBuckets i:nil="true" xmlns=""/></InvalidOperationException></Detail></Fault>